apps/docs/content/guides/database/prisma-troubleshooting.mdx

```diff
@@ -0,0 +1,247 @@
+---
+id: 'prisma-troubleshooting'
+title: 'Troubleshooting prisma errors'
+description: 'Prisma error troubleshooting'
+breadcrumb: 'ORM Quickstarts'
+---
+
+This guide addresses common Prisma errors that you might encounter while using Supabase.
+
+<Admonition type="note">
+
+A full list of errors can be found in [Prisma's official docs](https://www.prisma.io/docs/orm/reference/error-reference)
+
+</Admonition>
+
+## Understanding connection string parameters:
```

Comment:
Let's remove the colons from the headings, it looks a bit odd and the heading itself should be enough to indicate that it relates to the following text